change-notes/1.24/analysis-javascript.md

```diff
@@ -9,7 +9,10 @@
 * Imports with the `.js` extension can now be resolved to a TypeScript file,
   when the import refers to a file generated by TypeScript.
 
-* Imports that rely on path-mappings from a `tsconfig.json` file can now be resolved.
```

Comment:
Did you remove the `* Imports that rely on path-mappings from a tsconfig.json file can now be resolved.` line from the change-log?

Reply:
I refactored the change note a bit, but for some reason the diff is really weird when looking at the commit alone. If you look at the change notes under "Files changed" you should be able to see this line occur a few lines earlier.